diff -up hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_1000.rules.udev hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_1000.rules
--- hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_1000.rules.udev	2011-03-25 07:17:01.000000000 +0100
+++ hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_1000.rules	2011-05-04 21:11:42.000000000 +0200
@@ -3,9 +3,9 @@
 ACTION!="add", GOTO="hpmud_rules_end"
 
 # hp_laserjet_1000 for Suse 10.3 (has an hp parallel to usb dongle)
-SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", SYSFS{idVendor}=="03f0", SYSFS{idProduct}=="0517", PROGRAM="/bin/sh -c 'logger -p user.info loading hp_laserjet_1000 firmware $env{BUSNUM} $env{DEVNUM}'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s$env{BUSNUM}:$env{DEVNUM} &'"
+SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", ATTRS{idVendor}=="03f0", ATTRS{idProduct}=="0517", PROGRAM="/bin/sh -c 'logger -p user.info loading hp_laserjet_1000 firmware $env{BUSNUM} $env{DEVNUM}'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s$env{BUSNUM}:$env{DEVNUM} &'"
 
 # hp_laserjet_1000 (has an hp parallel to usb dongle)
-SUBSYSTEM=="usb_device", SYSFS{idVendor}=="03f0", SYSFS{idProduct}=="0517", PROGRAM="/bin/sh -c 'X=%k; X=$${X#usbdev}; B=$${X%%%%.*}; D=$${X#*.}; logger -p user.info loading hp_laserjet_1000 firmware $$B $$D; printf %%03i:%%03i $$B $$D'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s%c &'"
+SUBSYSTEM=="usb_device", ATTRS{idVendor}=="03f0", ATTRS{idProduct}=="0517", PROGRAM="/bin/sh -c 'X=%k; X=$${X#usbdev}; B=$${X%%%%.*}; D=$${X#*.}; logger -p user.info loading hp_laserjet_1000 firmware $$B $$D; printf %%03i:%%03i $$B $$D'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s%c &'"
 
 LABEL="hpmud_rules_end"
diff -up hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_1005_series.rules.udev hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_1005_series.rules
--- hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_1005_series.rules.udev	2011-03-25 07:17:01.000000000 +0100
+++ hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_1005_series.rules	2011-05-04 21:11:51.000000000 +0200
@@ -3,9 +3,9 @@
 ACTION!="add", GOTO="hpmud_rules_end"
 
 # hp_laserjet_1005 for Suse 10.3
-SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", SYSFS{idVendor}=="03f0", SYSFS{idProduct}=="1317", PROGRAM="/bin/sh -c 'logger -p user.info loading hp_laserjet_1005 firmware $env{BUSNUM} $env{DEVNUM}'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s$env{BUSNUM}:$env{DEVNUM} &'"
+SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", ATTRS{idVendor}=="03f0", ATTRS{idProduct}=="1317", PROGRAM="/bin/sh -c 'logger -p user.info loading hp_laserjet_1005 firmware $env{BUSNUM} $env{DEVNUM}'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s$env{BUSNUM}:$env{DEVNUM} &'"
 
 # hp_laserjet_1005
-SUBSYSTEM=="usb_device", SYSFS{idVendor}=="03f0", SYSFS{idProduct}=="1317", PROGRAM="/bin/sh -c 'X=%k; X=$${X#usbdev}; B=$${X%%%%.*}; D=$${X#*.}; logger -p user.info loading hp_laserjet_1005 firmware $$B $$D; printf %%03i:%%03i $$B $$D'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s%c &'"
+SUBSYSTEM=="usb_device", ATTRS{idVendor}=="03f0", ATTRS{idProduct}=="1317", PROGRAM="/bin/sh -c 'X=%k; X=$${X#usbdev}; B=$${X%%%%.*}; D=$${X#*.}; logger -p user.info loading hp_laserjet_1005 firmware $$B $$D; printf %%03i:%%03i $$B $$D'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s%c &'"
 
 LABEL="hpmud_rules_end"
diff -up hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_1018.rules.udev hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_1018.rules
--- hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_1018.rules.udev	2011-03-25 07:17:01.000000000 +0100
+++ hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_1018.rules	2011-05-04 21:12:11.000000000 +0200
@@ -3,9 +3,9 @@
 ACTION!="add", GOTO="hpmud_rules_end"
 
 # hp_laserjet_1018 for Suse 10.3
-SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", SYSFS{idVendor}=="03f0", SYSFS{idProduct}=="4117", PROGRAM="/bin/sh -c 'logger -p user.info loading hp_laserjet_1018 firmware $env{BUSNUM} $env{DEVNUM}'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s$env{BUSNUM}:$env{DEVNUM} &'"
+SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", ATTRS{idVendor}=="03f0", ATTRS{idProduct}=="4117", PROGRAM="/bin/sh -c 'logger -p user.info loading hp_laserjet_1018 firmware $env{BUSNUM} $env{DEVNUM}'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s$env{BUSNUM}:$env{DEVNUM} &'"
 
 # hp_laserjet_1018
-SUBSYSTEM=="usb_device", SYSFS{idVendor}=="03f0", SYSFS{idProduct}=="4117", PROGRAM="/bin/sh -c 'X=%k; X=$${X#usbdev}; B=$${X%%%%.*}; D=$${X#*.}; logger -p user.info loading hp_laserjet_1018 firmware $$B $$D; printf %%03i:%%03i $$B $$D'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s%c &'"
+SUBSYSTEM=="usb_device", ATTRS{idVendor}=="03f0", ATTRS{idProduct}=="4117", PROGRAM="/bin/sh -c 'X=%k; X=$${X#usbdev}; B=$${X%%%%.*}; D=$${X#*.}; logger -p user.info loading hp_laserjet_1018 firmware $$B $$D; printf %%03i:%%03i $$B $$D'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s%c &'"
 
 LABEL="hpmud_rules_end"
diff -up hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_1020.rules.udev hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_1020.rules
--- hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_1020.rules.udev	2011-03-25 07:17:01.000000000 +0100
+++ hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_1020.rules	2011-05-04 21:12:22.000000000 +0200
@@ -3,9 +3,9 @@
 ACTION!="add", GOTO="hpmud_rules_end"
 
 # hp_laserjet_1020 for Suse 10.3
-SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", SYSFS{idVendor}=="03f0", SYSFS{idProduct}=="2b17", PROGRAM="/bin/sh -c 'logger -p user.info loading hp_laserjet_1020 firmware $env{BUSNUM} $env{DEVNUM}'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s$env{BUSNUM}:$env{DEVNUM} &'"
+SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", ATTRS{idVendor}=="03f0", ATTRS{idProduct}=="2b17", PROGRAM="/bin/sh -c 'logger -p user.info loading hp_laserjet_1020 firmware $env{BUSNUM} $env{DEVNUM}'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s$env{BUSNUM}:$env{DEVNUM} &'"
 
 # hp_laserjet_1020
-SUBSYSTEM=="usb_device", SYSFS{idVendor}=="03f0", SYSFS{idProduct}=="2b17", PROGRAM="/bin/sh -c 'X=%k; X=$${X#usbdev}; B=$${X%%%%.*}; D=$${X#*.}; logger -p user.info loading hp_laserjet_1020 firmware $$B $$D; printf %%03i:%%03i $$B $$D'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s%c &'"
+SUBSYSTEM=="usb_device", ATTRS{idVendor}=="03f0", ATTRS{idProduct}=="2b17", PROGRAM="/bin/sh -c 'X=%k; X=$${X#usbdev}; B=$${X%%%%.*}; D=$${X#*.}; logger -p user.info loading hp_laserjet_1020 firmware $$B $$D; printf %%03i:%%03i $$B $$D'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s%c &'"
 
 LABEL="hpmud_rules_end"
diff -up hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_p1005.rules.udev hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_p1005.rules
--- hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_p1005.rules.udev	2011-03-25 07:17:01.000000000 +0100
+++ hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_p1005.rules	2011-05-04 21:12:30.000000000 +0200
@@ -3,9 +3,9 @@
 ACTION!="add", GOTO="hpmud_rules_end"
 
 # hp_laserjet_p1005 for Suse 10.3
-SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", SYSFS{idVendor}=="03f0", SYSFS{idProduct}=="3d17", PROGRAM="/bin/sh -c 'logger -p user.info loading hp_laserjet_p1005 firmware $env{BUSNUM} $env{DEVNUM}'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s$env{BUSNUM}:$env{DEVNUM} &'"
+SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", ATTRS{idVendor}=="03f0", ATTRS{idProduct}=="3d17", PROGRAM="/bin/sh -c 'logger -p user.info loading hp_laserjet_p1005 firmware $env{BUSNUM} $env{DEVNUM}'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s$env{BUSNUM}:$env{DEVNUM} &'"
 
 # hp_laserjet_p1005
-SUBSYSTEM=="usb_device", SYSFS{idVendor}=="03f0", SYSFS{idProduct}=="3d17", PROGRAM="/bin/sh -c 'X=%k; X=$${X#usbdev}; B=$${X%%%%.*}; D=$${X#*.}; logger -p user.info loading hp_laserjet_p1005 firmware $$B $$D; printf %%03i:%%03i $$B $$D'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s%c &'"
+SUBSYSTEM=="usb_device", ATTRS{idVendor}=="03f0", ATTRS{idProduct}=="3d17", PROGRAM="/bin/sh -c 'X=%k; X=$${X#usbdev}; B=$${X%%%%.*}; D=$${X#*.}; logger -p user.info loading hp_laserjet_p1005 firmware $$B $$D; printf %%03i:%%03i $$B $$D'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s%c &'"
 
 LABEL="hpmud_rules_end"
diff -up hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_p1006.rules.udev hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_p1006.rules
--- hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_p1006.rules.udev	2011-03-25 07:17:01.000000000 +0100
+++ hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_p1006.rules	2011-05-04 21:12:39.000000000 +0200
@@ -3,9 +3,9 @@
 ACTION!="add", GOTO="hpmud_rules_end"
 
 # hp_laserjet_p1006 for Suse 10.3
-SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", SYSFS{idVendor}=="03f0", SYSFS{idProduct}=="3e17", PROGRAM="/bin/sh -c 'logger -p user.info loading hp_laserjet_p1006 firmware $env{BUSNUM} $env{DEVNUM}'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s$env{BUSNUM}:$env{DEVNUM} &'"
+SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", ATTRS{idVendor}=="03f0", ATTRS{idProduct}=="3e17", PROGRAM="/bin/sh -c 'logger -p user.info loading hp_laserjet_p1006 firmware $env{BUSNUM} $env{DEVNUM}'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s$env{BUSNUM}:$env{DEVNUM} &'"
 
 # hp_laserjet_p1006
-SUBSYSTEM=="usb_device", SYSFS{idVendor}=="03f0", SYSFS{idProduct}=="3e17", PROGRAM="/bin/sh -c 'X=%k; X=$${X#usbdev}; B=$${X%%%%.*}; D=$${X#*.}; logger -p user.info loading hp_laserjet_p1006 firmware $$B $$D; printf %%03i:%%03i $$B $$D'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s%c &'"
+SUBSYSTEM=="usb_device", ATTRS{idVendor}=="03f0", ATTRS{idProduct}=="3e17", PROGRAM="/bin/sh -c 'X=%k; X=$${X#usbdev}; B=$${X%%%%.*}; D=$${X#*.}; logger -p user.info loading hp_laserjet_p1006 firmware $$B $$D; printf %%03i:%%03i $$B $$D'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s%c &'"
 
 LABEL="hpmud_rules_end"
diff -up hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_p1007.rules.udev hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_p1007.rules
--- hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_p1007.rules.udev	2011-03-25 07:17:01.000000000 +0100
+++ hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_p1007.rules	2011-05-04 21:12:47.000000000 +0200
@@ -3,9 +3,9 @@
 ACTION!="add", GOTO="hpmud_rules_end"
 
 # hp_laserjet_p1007 for Suse 10.3
-SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", SYSFS{idVendor}=="03f0", SYSFS{idProduct}=="4817", PROGRAM="/bin/sh -c 'logger -p user.info loading hp_laserjet_p1007 firmware $env{BUSNUM} $env{DEVNUM}'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s$env{BUSNUM}:$env{DEVNUM} &'"
+SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", ATTRS{idVendor}=="03f0", ATTRS{idProduct}=="4817", PROGRAM="/bin/sh -c 'logger -p user.info loading hp_laserjet_p1007 firmware $env{BUSNUM} $env{DEVNUM}'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s$env{BUSNUM}:$env{DEVNUM} &'"
 
 # hp_laserjet_p1007
-SUBSYSTEM=="usb_device", SYSFS{idVendor}=="03f0", SYSFS{idProduct}=="4817", PROGRAM="/bin/sh -c 'X=%k; X=$${X#usbdev}; B=$${X%%%%.*}; D=$${X#*.}; logger -p user.info loading hp_laserjet_p1007 firmware $$B $$D; printf %%03i:%%03i $$B $$D'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s%c &'"
+SUBSYSTEM=="usb_device", ATTRS{idVendor}=="03f0", ATTRS{idProduct}=="4817", PROGRAM="/bin/sh -c 'X=%k; X=$${X#usbdev}; B=$${X%%%%.*}; D=$${X#*.}; logger -p user.info loading hp_laserjet_p1007 firmware $$B $$D; printf %%03i:%%03i $$B $$D'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s%c &'"
 
 LABEL="hpmud_rules_end"
diff -up hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_p1008.rules.udev hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_p1008.rules
--- hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_p1008.rules.udev	2011-03-25 07:17:01.000000000 +0100
+++ hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_p1008.rules	2011-05-04 21:12:56.000000000 +0200
@@ -3,9 +3,9 @@
 ACTION!="add", GOTO="hpmud_rules_end"
 
 # hp_laserjet_p1008 for Suse 10.3
-SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", SYSFS{idVendor}=="03f0", SYSFS{idProduct}=="4917", PROGRAM="/bin/sh -c 'logger -p user.info loading hp_laserjet_p1008 firmware $env{BUSNUM} $env{DEVNUM}'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s$env{BUSNUM}:$env{DEVNUM} &'"
+SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", ATTRS{idVendor}=="03f0", ATTRS{idProduct}=="4917", PROGRAM="/bin/sh -c 'logger -p user.info loading hp_laserjet_p1008 firmware $env{BUSNUM} $env{DEVNUM}'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s$env{BUSNUM}:$env{DEVNUM} &'"
 
 # hp_laserjet_p1008
-SUBSYSTEM=="usb_device", SYSFS{idVendor}=="03f0", SYSFS{idProduct}=="4917", PROGRAM="/bin/sh -c 'X=%k; X=$${X#usbdev}; B=$${X%%%%.*}; D=$${X#*.}; logger -p user.info loading hp_laserjet_p1008 firmware $$B $$D; printf %%03i:%%03i $$B $$D'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s%c &'"
+SUBSYSTEM=="usb_device", ATTRS{idVendor}=="03f0", ATTRS{idProduct}=="4917", PROGRAM="/bin/sh -c 'X=%k; X=$${X#usbdev}; B=$${X%%%%.*}; D=$${X#*.}; logger -p user.info loading hp_laserjet_p1008 firmware $$B $$D; printf %%03i:%%03i $$B $$D'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s%c &'"
 
 LABEL="hpmud_rules_end"
diff -up hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_p1505.rules.udev hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_p1505.rules
--- hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_p1505.rules.udev	2011-03-25 07:17:01.000000000 +0100
+++ hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_p1505.rules	2011-05-04 21:13:03.000000000 +0200
@@ -3,9 +3,9 @@
 ACTION!="add", GOTO="hpmud_rules_end"
 
 # hp_laserjet_p1505 for Suse 10.3
-SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", SYSFS{idVendor}=="03f0", SYSFS{idProduct}=="3f17", PROGRAM="/bin/sh -c 'logger -p user.info loading hp_laserjet_p1505 firmware $env{BUSNUM} $env{DEVNUM}'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s$env{BUSNUM}:$env{DEVNUM} &'"
+SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", ATTRS{idVendor}=="03f0", ATTRS{idProduct}=="3f17", PROGRAM="/bin/sh -c 'logger -p user.info loading hp_laserjet_p1505 firmware $env{BUSNUM} $env{DEVNUM}'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s$env{BUSNUM}:$env{DEVNUM} &'"
 
 # hp_laserjet_p1505
-SUBSYSTEM=="usb_device", SYSFS{idVendor}=="03f0", SYSFS{idProduct}=="3f17", PROGRAM="/bin/sh -c 'X=%k; X=$${X#usbdev}; B=$${X%%%%.*}; D=$${X#*.}; logger -p user.info loading hp_laserjet_p1505 firmware $$B $$D; printf %%03i:%%03i $$B $$D'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s%c &'"
+SUBSYSTEM=="usb_device", ATTRS{idVendor}=="03f0", ATTRS{idProduct}=="3f17", PROGRAM="/bin/sh -c 'X=%k; X=$${X#usbdev}; B=$${X%%%%.*}; D=$${X#*.}; logger -p user.info loading hp_laserjet_p1505 firmware $$B $$D; printf %%03i:%%03i $$B $$D'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s%c &'"
 
 LABEL="hpmud_rules_end"
diff -up hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_professional_p1102.rules.udev hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_professional_p1102.rules
--- hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_professional_p1102.rules.udev	2011-03-25 07:17:01.000000000 +0100
+++ hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_professional_p1102.rules	2011-05-04 21:13:24.000000000 +0200
@@ -3,9 +3,9 @@
 ACTION!="add", GOTO="hpmud_rules_end"
 
 # hp_laserjet_professional_p1102 for Suse 10.3
-SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", SYSFS{idVendor}=="03f0", SYSFS{idProduct}=="002a", PROGRAM="/bin/sh -c 'logger -p user.info loading hp_laserjet_professional_p1102 firmware $env{BUSNUM} $env{DEVNUM}'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s$env{BUSNUM}:$env{DEVNUM} &'"
+SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", ATTRS{idVendor}=="03f0", ATTRS{idProduct}=="002a", PROGRAM="/bin/sh -c 'logger -p user.info loading hp_laserjet_professional_p1102 firmware $env{BUSNUM} $env{DEVNUM}'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s$env{BUSNUM}:$env{DEVNUM} &'"
 
 # hp_laserjet_professional_p1102
-SUBSYSTEM=="usb_device", SYSFS{idVendor}=="03f0", SYSFS{idProduct}=="002a", PROGRAM="/bin/sh -c 'X=%k; X=$${X#usbdev}; B=$${X%%%%.*}; D=$${X#*.}; logger -p user.info loading hp_laserjet_professional_p1102 firmware $$B $$D; printf %%03i:%%03i $$B $$D'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s%c &'"
+SUBSYSTEM=="usb_device", ATTRS{idVendor}=="03f0", ATTRS{idProduct}=="002a", PROGRAM="/bin/sh -c 'X=%k; X=$${X#usbdev}; B=$${X%%%%.*}; D=$${X#*.}; logger -p user.info loading hp_laserjet_professional_p1102 firmware $$B $$D; printf %%03i:%%03i $$B $$D'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s%c &'"
 
 LABEL="hpmud_rules_end"
diff -up hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_professional_p1102w.rules.udev hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_professional_p1102w.rules
--- hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_professional_p1102w.rules.udev	2011-03-25 07:17:01.000000000 +0100
+++ hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_professional_p1102w.rules	2011-05-04 21:13:33.000000000 +0200
@@ -3,9 +3,9 @@
 ACTION!="add", GOTO="hpmud_rules_end"
 
 # hp_laserjet_professional_p1102w for Suse 10.3
-SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", SYSFS{idVendor}=="03f0", SYSFS{idProduct}=="002a", PROGRAM="/bin/sh -c 'logger -p user.info loading hp_laserjet_professional_p1102w firmware $env{BUSNUM} $env{DEVNUM}'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s$env{BUSNUM}:$env{DEVNUM} &'"
+SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", ATTRS{idVendor}=="03f0", ATTRS{idProduct}=="002a", PROGRAM="/bin/sh -c 'logger -p user.info loading hp_laserjet_professional_p1102w firmware $env{BUSNUM} $env{DEVNUM}'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s$env{BUSNUM}:$env{DEVNUM} &'"
 
 # hp_laserjet_professional_p1102w
-SUBSYSTEM=="usb_device", SYSFS{idVendor}=="03f0", SYSFS{idProduct}=="002a", PROGRAM="/bin/sh -c 'X=%k; X=$${X#usbdev}; B=$${X%%%%.*}; D=$${X#*.}; logger -p user.info loading hp_laserjet_professional_p1102w firmware $$B $$D; printf %%03i:%%03i $$B $$D'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s%c &'"
+SUBSYSTEM=="usb_device", ATTRS{idVendor}=="03f0", ATTRS{idProduct}=="002a", PROGRAM="/bin/sh -c 'X=%k; X=$${X#usbdev}; B=$${X%%%%.*}; D=$${X#*.}; logger -p user.info loading hp_laserjet_professional_p1102w firmware $$B $$D; printf %%03i:%%03i $$B $$D'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s%c &'"
 
 LABEL="hpmud_rules_end"
diff -up hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_professional_p1566.rules.udev hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_professional_p1566.rules
--- hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_professional_p1566.rules.udev	2011-03-25 07:17:01.000000000 +0100
+++ hplip-plugins-3.11.3a-1.x86_64/86-hpmud-hp_laserjet_professional_p1566.rules	2011-05-04 21:13:38.000000000 +0200
@@ -3,9 +3,9 @@
 ACTION!="add", GOTO="hpmud_rules_end"
 
 # hp_laserjet_professional_p1566 for Suse 10.3
-SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", SYSFS{idVendor}=="03f0", SYSFS{idProduct}=="092a", PROGRAM="/bin/sh -c 'logger -p user.info loading hp_laserjet_professional_p1566 firmware $env{BUSNUM} $env{DEVNUM}'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s$env{BUSNUM}:$env{DEVNUM} &'"
+SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", ATTRS{idVendor}=="03f0", ATTRS{idProduct}=="092a", PROGRAM="/bin/sh -c 'logger -p user.info loading hp_laserjet_professional_p1566 firmware $env{BUSNUM} $env{DEVNUM}'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s$env{BUSNUM}:$env{DEVNUM} &'"
 
 # hp_laserjet_professional_p1566
-SUBSYSTEM=="usb_device", SYSFS{idVendor}=="03f0", SYSFS{idProduct}=="092a", PROGRAM="/bin/sh -c 'X=%k; X=$${X#usbdev}; B=$${X%%%%.*}; D=$${X#*.}; logger -p user.info loading hp_laserjet_professional_p1566 firmware $$B $$D; printf %%03i:%%03i $$B $$D'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s%c &'"
+SUBSYSTEM=="usb_device", ATTRS{idVendor}=="03f0", ATTRS{idProduct}=="092a", PROGRAM="/bin/sh -c 'X=%k; X=$${X#usbdev}; B=$${X%%%%.*}; D=$${X#*.}; logger -p user.info loading hp_laserjet_professional_p1566 firmware $$B $$D; printf %%03i:%%03i $$B $$D'", RUN+="/bin/sh -c '/usr/bin/hp-firmware -y3 -s%c &'"
 
 LABEL="hpmud_rules_end"
